test/new_tests/test_get_cdtctx_base64.py

```diff
@@ -101,7 +98,7 @@ def test_get_cdtctxb64_with_correct_parameters(self):
         Invoke get_cdtctx_base64() with correct arguments
         """
         policy = {}
-        bs_b4_cdt = self.as_connection.get_cdtctx_base64({"ctx": ctx_list_index})
+        bs_b4_cdt = self.as_connection.get_cdtctx_base64([cdt_ctx.cdt_ctx_list_index(0)])
```

Comment:
Why is this using index 0 and not the whole list. The previous version worked on the CTX list value wrapped in the Map key 'ctx'